upnext/tracks.py

```diff
@@ -49,8 +49,9 @@ def add_to_playlist(track_uri, party, added_by_user):
     # Adding in DB
     track_artist = unicodedata.normalize('NFKD', track_info['artist']).encode('ascii', 'ignore')
     track_title = unicodedata.normalize('NFKD', track_info['song_title']).encode('ascii', 'ignore')
+    track_preview = track_info['preview']
 
-    track = Track(track_title=track_title, artist=track_artist, uri=track_uri, party=party, added_by=added_by_user)
+    track = Track(track_title=track_title, artist=track_artist, uri=track_uri, party=party, preview=track_preview, added_by=added_by_user)
     track.save()
```

Comment:
This isn't the only place we're making a new track... also in `create_party`, right? I can change that in my PR once this ships, but until then loading from playlists is going to be very broken.